test/integration/requests_flow_test.rb

```diff
@@ -9,47 +9,34 @@ class RequestsFlowTest < ActionDispatch::IntegrationTest
     click_link classrooms(:two).name
   end
 
-  test "a student can add a question" do
-    fill_in :request_question, with: "I has a question!"
+  test "student can request help and advance the queue" do
+    fill_in :request_question, with: "I have a question to add!"
     click_button 'Ask Question'
+    assert page.has_content?("I have a question to add!")
 
-    assert page.find('#requests-list').has_content?("I has a question!")
-  end
-
-  test "student can mark question as being-helped" do
-    fill_in :request_question, with: "I has another question!"
-    click_button 'Ask Question'
-
-    request_id = Request.all.last.id
-    find("#request#{request_id}").find(".request-toggle").click
+    within(all(".request-item").last) { click_link 'Being Helped?' }
+    assert all(".request-item").last.has_content?("Being Helped")
+    sleep(1)
 
-    assert find("#request#{request_id}").has_content?("Being Helped")
+    within(all(".request-item").last) { click_link 'Done?' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.